chalicelib_smaht/checks/ecs_checks.py

```diff
@@ -46,17 +46,17 @@ def update_ecs_application_versions(connection, **kwargs):
     cluster_name = kwargs.get('cluster_name')
     cluster_arns = client.list_ecs_clusters()
     if not cluster_name:
-        cgap_candidate = list(filter(lambda arn: 'cgap' in arn.lower(), cluster_arns))
-        if not cgap_candidate:
+        smaht_candidate = list(filter(lambda arn: 'smaht' in arn.lower(), cluster_arns))
```

Comment:
Assuming EnvUtils has been initialized, which I assume by here it is, should this be using `EnvUtils.app_name()`?  Maybe there is comfort in seeing the wired name, but it means a lot of specific updates for forks like this.